src/index.ts

```diff
@@ -137,6 +138,10 @@ class MultinetAPI {
     return this.client.delete(`/workspaces/${workspace}`);
   }
 
+  public renameWorkspace(workspace: string, name: string): AxiosPromise {
+    return this.client.axios.put(`workspaces/${workspace}/name`, null, { params: { name }});
+  }
+
```

Comment:
In general, when you mix in substantive changes like this, it becomes harder to understand the changesets. In particular, in this instance, neither the commit message nor the PR description hint at this feature being added. I'd go so far as to say this change should have appeared in a separate PR from the one that introduces the private-public switch and the associated code simplifications.

In this case, if you update the PR description, I'll be happy. But in the future, please make sure to split up even simple PRs like this into logical commits, and make sure to put substantive features in their own PRs.

Reply:
You're right, that was really part of the other branch I was working on, and I just remember to remove it.